app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+
+  def create
+    user = Person.from_omniauth(request.env["omniauth.auth"])
+    session[:user_id] = user.id
+    redirect_to '/'
+  end
+
+  def destroy
+    session[:user_id] = nil
+    redirect_to root_path
+  end
+
+  protected
+
+  def auth_hash
+    request.env['omniauth.auth']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.